### DISCUSSION TITLE
Missing APK

### DISCUSSION THREAD

**Discussion by slcw** [tutao/tutanota#8093] [General]
Any idea why the APK for v253.241203.1 has been removed from the repo? It was there, I installed it, then it disappeared. The result is Obtainium now thinks v253.241126.2 is the latest release, and is putting up annoying update messages, even though I can't update to an older version that's installed. Obviously, this isn't a major problem, or one that even needs to be remedied, but I am curious why it was removed.

**Comment by Gitchick57** [comment]
I didn't even know there was an update.  It only shows up in Playstore which I won't download.



Dec 6, 2024, 7:51 PM by ***@***.***:

**Reply by slcw** [reply]
The last update was on 11/26, and it updated v253.241126.2 to v253.241203.1. Then a few days ago the v253.241203.1 APK disappeared from the repo leaving v253.241126.2 as the latest available release. The result is that Obtainium erroneously believes v253.241126.2 is the latest release and is prompting me to update. But obviously I'm not downgrading to v253.241126.2 just make Obtainium happy. I'll just deal with the update notification until the next new update is released. I'm just curious why Tuta pulled the v253.241203.1 APK from the repo.🤷‍♂️

**Reply by charlag** [reply]
Hey, there could not be a v253.241203.1 release in November since it was built on 3rd of December, like the version implies.

**Reply by slcw** [reply]
My update log shows it was updated to v253.241203.1 on Nov 26, 10:21:06. That Dec 3 date you mentioned lines up with the date they removed v253.241203.1 from the repo.

My dates are based on the actual timestamps logged for every update, along with the corresponding information on the specific apk used to execute the update, so I'm pretty confident on the time-line.

**Comment by Gitchick57** [comment]
Interesting... I will wait for the next release as well.  Thanks for your response.



Dec 7, 2024, 12:08 AM by ***@***.***:
